src/main/groovy/util/StockMovementContext.groovy

```diff
@@ -0,0 +1,58 @@
+package util
+
+import org.pih.warehouse.api.StockMovementDirection
+import org.pih.warehouse.auth.AuthService
+import org.pih.warehouse.core.Location
+import org.pih.warehouse.order.Order
+import org.pih.warehouse.requisition.Requisition
+import org.pih.warehouse.requisition.RequisitionSourceType
+import org.pih.warehouse.shipping.Shipment
+
+class StockMovementContext {
```

Comment:
Let's call this class StockMovementStatusContext since it's really the context we are using in order to resolve a status, not a stock movement.